easybuild/easyconfigs/g/Gctf/Gctf-wrapper.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Wrap around Gctf to load the required toolchain so it can use the
+# expected version of CUDA.
+
+# Save the path to the install tree
+gctf_base=$EBROOTGCTF
+
+module purge > /dev/null 2>&1
```

Comment:
@akesandgren Why not only unload `CUDA`, should be safer than `purge`?

Reply:
For Gctf it makes sense, I just noticed that it depends on libstdc++6 so it needs a GCCcore too to be safe against very minimal OS:es.
(On the other hand libstdc++.so.6 is a vital part of all Linux distros that are still used)